makefiles/Makefile.inc

```diff
@@ -29,6 +29,11 @@
 
 # Backwards-compatibility wrapper for people using
 # "include $COCOTB/makefiles/Makefile.inc" in their Makefile.
+
+$(warning 'Deprecated cocotb file structure detected. Please install cocotb \
+(see documentation) and set "COCOTB:=$$(shell cocotb-config --share)" \
+or "include $$(shell cocotb-config --makefile)/Makefile.sim" in Makefiles')
```

Comment:
```suggestion
# Make sure cocotb is installed
COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
ifeq ($(COCOTB_CONFIG_CMD),)
    $(error "Cocotb not installed (cocotb-config not found). Please install cocotb (see documentation for details).")
endif

$(warning 'Deprecated cocotb file structure detected. Using "include $$(shell cocotb-config --makefile)/Makefile.sim" instead')
include $(shell cocotb-config --makefile)/Makefile.sim
```

Reply:
Missing the "is" in the suggestion: ``$(error "Cocotb is not installed``